packages/core/src/common/audioAlertManager.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  Annotations,
+  breachedThreshold,
+  DataStream as SynchroChartsDataStream,
+  getThresholds,
+  MinimalViewPortConfig,
+  Threshold,
+} from '@synchro-charts/core';
+import { DataStream } from '../data-module/types';
+
+import { AudioAlert } from './audioAlert';
+import { liveDataTimeBuffer } from './constants';
+import { getVisibleData } from './dataFilters';
+import { getDataPoints } from './getDataPoints';
+import { isNumberDataStream } from './predicates';
+
+// returns true if viewport is in Live Mode or if viewport.end is past the current time and date
+export const isLiveData = (viewport: MinimalViewPortConfig): boolean => {
+  // duration in viewport if in live mode
+  if ('duration' in viewport) {
+    return true;
+  } else {
+    // if the viewport is past the current time then we expect audio alerts to behave as if we're in live mode
+    const endTimestamp = typeof viewport.end === 'string' ? new Date(viewport.end).getTime() : viewport.end.getTime();
+    return endTimestamp + liveDataTimeBuffer > Date.now();
```

Comment:
nit: would be nice to have a small comment explaining why we choose 1.5 seconds specifically.